docs/branch_mgmt.md

```diff
@@ -88,7 +88,7 @@ A scratch view will be opened, showing the diff between the selected branch and
 
 #### Diff history of selected and active branches (`H`)
 
-A scratch view will be opened, showing the commit diff between the selected branch and the active branch.
+A scratch view will be opened, showing the commit diff between the selected branch and the active branch. To cherry-pick a commit, move one courser over the commit you want to cherry-pick and press `c`. The commit might not appear at once then refresh the view  by pressing `r`.
```

Comment:
than it will be easier to find for everyone. I don't think there is anyone that would hate us if the change the keybindings other then for any of the interfaces.

Should that we a separate PR of in this one. I will do it for the graph view too.